test/utils/writeTypingsHelper.js

```diff
@@ -26,7 +26,11 @@ module.exports = {
     });
     return dir;
   },
-  dynamic: (rootTestPath, context, { elementsMap, widgets, dependencies }) => {
+  dynamic: (
+    rootTestPath,
+    context,
+    { elementsMap, widgets, dependencies, jswFiles }
```

Comment:
should we just get a list of {name, content} objects and write them to files without knowing their categorization?

Reply:
Done 👍
Turned out real nice I think :)